.github/workflows/profiler-pipeline.yml

```
@@ -281,8 +279,7 @@ jobs:
       - name: Build Native Profiler Engine
         run: |
           CXX=clang++ CC=clang cmake -S profiler -B __build -DRUN_UBSAN=ON
-          cd __build
-          make
+          cmake --build __build --config Release
```

Should this have the parallel flag too?
```suggestion
          cmake --build __build --config Release --parallel
```

---

I do not know why, but currently the `UBSAN` job is failing when compiling in parallel :/
I deactivated it to confirm.